locales/bn/bn.json

```diff
@@ -346,15 +346,15 @@
     "Mayotte": "মায়োট",
     "Mexico": "মেক্সিকো",
     "Micronesia, Federated States Of": "মাইক্রোনেশিয়া",
-    "Micronesia, Federated States of": "Micronesia, Federated States of",
+    "Micronesia, Federated States of": "মাইক্রোনেশিয়া, ফেডারেটেড স্টেট অফ",
```

Comment:
Why two different translations ?

Reply:
The [main source](https://github.com/Laravel-Lang/lang/blob/master/locales/bn/bn.json#L348) has duplicate key, I'll delete duplicate